### PR TITLE
feat: Add resolution hour to grid loss produced

### PR DESCRIPTION
**Pull request by johevemi**
# Description
GridLossResultProducedV1 add hour to resolution enum

The events sent wont have hour before this PR https://github.com/Energinet-DataHub/opengeh-wholesale/pull/2191 is in main

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [x] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)
- [ ] Documentation has been updated
- [ ] The integration [event catalog](https://energinet.atlassian.net/wiki/spaces/D3/pages/555581556/Event+catalog) has been updated
- [ ] C4 diagrams have been updated and Team Outlaws has been informed about relevant changes

### CODE REVIEW

File: source/dotnet/wholesale-integration-events-package/Contracts/Contracts.csproj [hunk 1]
```diff
@@ -21,7 +21,7 @@ limitations under the License.
 
   <PropertyGroup>
     <PackageId>Energinet.DataHub.Wholesale.Contracts</PackageId>
-    <PackageVersion>9.1.2$(VersionSuffix)</PackageVersion>
+    <PackageVersion>9.1.3$(VersionSuffix)</PackageVersion>
```
Comment (JesperGomes) [inline source/dotnet/wholesale-integration-events-package/Contracts/Contracts.csproj:24]: 9.2.0?

Comment (johevemi) [reply source/dotnet/wholesale-integration-events-package/Contracts/Contracts.csproj:24]: why

Comment (johevemi) [reply source/dotnet/wholesale-integration-events-package/Contracts/Contracts.csproj:24]: adding a column is a minor change

Comment (johevemi) [reply source/dotnet/wholesale-integration-events-package/Contracts/Contracts.csproj:24]: will change to 9.2.0 i misunderstood what minor was

**Review by JesperGomes — APPROVED** [review]

**Pull request merged**